Proofreading-OV-Runtime (#11658)
openvinotoolkit/openvino: tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware/README.md

# AccuracyAwareQuantization Algorithm {#accuracy_aware_README}

## Introduction

> AccuracyAwareQuantization Algorithm purpouse is to accurate quantization and allowing the model's accuracy to stay within the 

Author: ```suggestion
The purpose of AccuracyAwareQuantization Algorithm is an accurate quantization and keeping the accuracy of the model to stay within the 
```

pre-defined range. In comparison to [DefaultQuantization](../default/README.md) algorithm this may cause a 
degradation in performance because some layers can be reverted back to the original precision.

## Parameters

> Since the [DefaultQuantization](../default/README.md) algorithm is used as an initialization, all its parameters are also valid and can be specified. Below is an example of lthe definition of the `AccuracyAwareQuantization` method and its parameters:

Author: ```suggestion
Since the [DefaultQuantization](../default/README.md) algorithm is used as an initialization, all its parameters are also valid and can be specified. Below is an example definition of the `AccuracyAwareQuantization` method and its parameters:
```

}
```

Below are the descriptions of AccuracyAwareQuantization-specific parameters:
- `"ranking_subset_size"` - size of a subset that is used to rank layers by their contribution to the accuracy drop. 

> Default value is `300`, and more samples it has the better ranking you have, potentially.

Author: ```suggestion
Default value is `300`, and more samples it has the better ranking, potentially.
```

> - `"max_iter_num"` - maximum number of iterations of the algorithm. In other words, maximum number of layers that may

Author: ```suggestion
- `"max_iter_num"` - the maximum number of iterations of the algorithm. In other words, the maximum number of layers that may
```

>  be reverted back to floating-point precision. By default it is limited by the overall number of quantized layers.

Author: ```suggestion
 be reverted back to floating-point precision. By default, it is limited by the overall number of quantized layers.
```

- `"maximal_drop"` - maximum accuracy drop which has to be achieved after the quantization. Default value is `0.01` (1%).

> - `"drop_type"` - drop type of the accuracy metric: 

Author: ```suggestion
- `"drop_type"` - a drop type of the accuracy metric: 
```

>     - `"absolute"` - (default) absolute drop with respect to the results of the full-precision model.

Author: ```suggestion
    - `"absolute"` - the (default) absolute drop with respect to the results of the full-precision model.
```

    - `"relative"` - relative to the results of the full-precision model.
- `"use_prev_if_drop_increase"` - whether to use network snapshot from the previous iteration of in case if drop 
increases. Default value is `True`.

> - `"base_algorithm"` - name of the algorithm that is used to quantize model at the beginning. Default value is 

Author: ```suggestion
- `"base_algorithm"` - name of the algorithm that is used to quantize a model at the beginning. The default value is 
```

    "DefaultQuantization".
- `"convert_to_mixed_preset"` - whether to convert the model to "mixed" mode if the accuracy criteria for the model
 quantized with "performance" preset are not satisfied. This option can help to reduce number of layers that are reverted
 to floating-point precision. Keep in mind that this is an **experimental** feature.
- `"metrics"` - optional list of metrics that are taken into account during optimization. It consists of tuples with the 
following parameters:
    - `"name"` - name of the metric to optimize.

>     - `"baseline_value"` - (optional parameter) baseline metric value of the original model. The validations on

Author: ```suggestion
    - `"baseline_value"` - (optional parameter) a baseline metric value of the original model. The validations on
```

>     the whole validation will be initiated in the beginning if nothing specified.

Author: ```suggestion
    The validation will be initiated entirely in the beginning if nothing specified.
```

> - `"metric_subset_ratio"` - part of the validation set that is used to compare original full-precision and 

Author: ```suggestion
- `"metric_subset_ratio"` - a part of the validation set that is used to compare original full-precision and 
```

> fully quantized models when creating ranking subset in case of predefined metric values of the original model.

Author: ```suggestion
fully quantized models when creating a ranking subset in case of predefined metric values of the original model.
```

Default value is `0.5`.

> - `"tune_hyperparams"` - enables quantization parameters tuning as a preliminary step before reverting layers back

Author: ```suggestion
- `"tune_hyperparams"` - enables tuning of quantization parameters as a preliminary step before reverting layers
```

> to the floating-point precision. It can bring additional performance and accuracy boost but increase overall 

Author: ```suggestion
to the floating-point precision. It can bring an additional boost in performance and accuracy, at the cost of increased overall 
```
